README.md

```diff
@@ -1,94 +1,92 @@
-# Ibus-typing-booster
+ibus-typing-booster
+===================
 
 Faster typing by context sensitive completion.
 
-## Introduction
+Introduction
+------------
 
 Ibus-typing-booster is a completion input method to speedup typing.
 
-The project was started in 2010 for Fedora 15. The original purpose was to make typing of
-Indic languages easier and faster by providing completion and spell checking suggestions.
+The project was started in 2010 for Fedora 15. The original purpose was to make typing of Indic languages easier and faster by providing completion and spell checking suggestions.
+
+Originally it was forked from ibus-table whose developer was Yu Yuwei acevery@gmail.com, with contributions from Caius “kaio” Chance me@kaio.net.
+
+Since then ibus-typing-booster has been improved to support many other languages as well (most languages except Chinese and Japanese are supported).
+
+Recently the capapility to type different languages at the same time without having to switch between languages has been added.
+
+Developers
+----------
+- Mike FABIAN mfabian@redhat.com<br>
+- Anish Patil anish.developer@gmail.com
+
+Features
+--------
+- Context sensitive completions.<br>
+- Learns from user input.<br>
+- Can be trained by supplying files containing typical user input.<br>
+- If available, hunspell and hunspell dictionaries will also be used to provide not only<br>
+  completion but also spellchecking suggestions (But ibus-typing-booster works also<br>
+  without hunspell by learning only from user input).<br>
+- Can be used with any keyboard layout.<br>
+- Almost all input methods supplied by libm17n are supported (including the inscript2<br>
+  input methods).<br>
+- Several input methods and languages can be used at the same time without switching.<br>
+- Predicts Unicode symbols and emoji as well.
+
+Requirements
+------------
+- Some packages are requied if you are building from source:
+```
+    ┏━━━━━━━━━━━━━━━━━━━┯━━━━━━━━━┓
+    ┃      Package      │ Version ┃
```

Comment:
Your build  requirements  table is only for Ubuntu, it is quite different for Fedora,  openSUSE, ...

http://mike-fabian.github.io/ibus-typing-booster/development.html

tries to list the build requirements for different distributions, I would rather update the Ubuntu section there if
you think something is missing there.

Reply:
Ok, I see.